helm/dagster/schema/schema_tests/test_celery_queues.py

```diff
@@ -307,3 +307,35 @@ def test_scheduler_name(deployment_template: HelmTemplate):
 
     deployment = celery_queue_deployments[0]
     assert deployment.spec.template.spec.scheduler_name == "custom"
+
+
+def test_check_db_container_toggle(deployment_template: HelmTemplate):
+    helm_values = DagsterHelmValues.construct(
+        runLauncher=RunLauncher(
+            type=RunLauncherType.CELERY,
+            config=RunLauncherConfig(
+                celeryK8sRunLauncher=CeleryK8sRunLauncherConfig.construct(
+                    checkDbReadyInitContainer=False
+                )
+            ),
+        )
+    )
+    [daemon_deployment] = deployment_template.render(helm_values)
+    assert daemon_deployment.spec.template.spec.init_containers is None or "check-db-ready" not in [
```

Comment:
The first `is None` check is not strictly necessary for all tests, but default templates have initContainers and some don't. The shortcircuit makes it so the tests which _do_ have an initContainer by default will not start failing if that container is removed.